lib/macmillan/utils/rspec/matchers/be_valid_mosaic.rb

```diff
@@ -0,0 +1,43 @@
+require 'rspec/expectations'
+
+### Example:
+### it { is_expected.to be_valid_mosaic }
+#
+### example output:
+#
+### Failure/Error: it { is_expected.to be_valid_mosaic }
+### Missing required property: template in /layout
+
+RSpec::Matchers.define :be_valid_mosaic do
```

Comment:
being public isn't a bit too connected to a closed source project?

Maybe doesn't make much sense to have this repo open btw...